python/tests/kernel/test_run_kernel.py

```diff
@@ -15,6 +15,7 @@
 import pytest
 
 list_err_msg = 'does not yet support returning `list` from entry-point kernels'
+ALLOWED_TYPES_IN_A_DATACLASS = [int, float, bool]
```

Comment:
Import this from the kernel module. No need to define it in two different places and incur the possibility of these being different.